website/project/views/contributor.py

```diff
@@ -633,7 +633,6 @@ def claim_user_form(auth, **kwargs):
             # Redirect to CAS and authenticate the user with a verification key.
```

Comment:
@icereval Looks like the helper is already there, I can just used `generate_confirm_token` and `generate_claim_token` for our purpose (with token length).

@abought 30-charactor is OK but `62^30 <= 64^32 = 2^160`. I am for 40-character string.